lib/medtronic600/NGPHistoryParser.js

```diff
@@ -927,10 +925,12 @@ class NGPHistoryParser {
           {
             const targets = event.bgTargets();
             // Because we read the initial pump settings in mg/dL, always convert settings changes.
+            // When we confirm that we can determine pump BG units correctly in
+            // DeviceCharacteristicsResponse:units(), we can remove the conversion here.
             if (event.units === NGPUtil.NGPConstants.BG_UNITS.MMOL_L) {
               for (const target of targets) {
-                target.high = Math.ceil(target.high * 18);
-                target.low = Math.ceil(target.low * 18);
+                target.high = parseFloat((target.high * 18.01559).toFixed(2));
```

Comment:
I forgot to mention that there's now a common function to use: `convertBackToMmol` in `commonFunctions.js`.